articles/application-insights/app-insights-tutorial-dashboards.md

```diff
@@ -84,7 +84,7 @@ The **Metrics** panel allows you to graph a metric collected by Application Insi
 
 3. At the top of the screen, click **View dashboard** to return to your dashboard.
 
-4. The Overview Timeline is now added to your dashboard.  Click and drag it into position and then click **Done customizing** and then **Publish changes**. 
+4. You now have the Metrics chart added to your dashboard. Click and drag it into position and then click **Done customizing** and then **Publish changes**. 
```

Comment:
@srcsakthivel Thank you for your suggestion. I am not sure I completely understand what issue replacing "Overview Timeline" with "Metrics chart" addresses? If the issue is the lack of specificity as to what the "Overview Timeline" is, rather than dropping the name of the chart and replacing it with Metrics I would suggest modifying your pull request to:

4. The Overview Timeline Metrics chart is now added to your dashboard.  Click and drag it into position and then click **Done customizing** and then **Publish changes**.

Reply:
@srcsakthivel Perfect, if you make the above changes to the pull request I would be happy to sign off and get it merged. Thanks!

Reply:
"Overview Timeline" is already defined chart in Application insight Overview. As "Overview Timeline" is explained just above the "Metrics Charting", I felt calling it out as Metrics Chart would make more sense and less confusing.